DEVELOPMENT.md

```diff
@@ -2,6 +2,20 @@ Tips for developers
 -------------------
 It's dangerous to go alone. Take these tips in case you need to fit Indico to your particular needs.
 
+## Running a virtual machine
+Install [vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/), install
+[vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest) plugin with `vagrant plugin install vagrant-vbguest`.
+Simply call `vagrant up` to get your virtual machine running and `vagrant ssh` to jump into the virtual machine.
+
+Next steps:
+```
+indico admin user_create -a
```

Comment:
`echo -e "foo\nbar\n" | indico admin user_create -a` might work for this.

However, the bootstrap wizard is shown when you access Indico in a browser with no users in the database. So it would not require running any commandline tools interactively.